dask/dataframe/io/tests/test_parquet.py

```diff
@@ -938,6 +938,60 @@ def test_to_parquet_default_writes_nulls(tmpdir):
     assert table[1].null_count == 2
 
 
+def test_to_parquet_consistent_schema_partition_pyarrow(tmpdir, engine):
+    check_fastparquet()
+    check_pyarrow()
```

Comment:
My guess is that we don't want these checks here.  My guess is that this is handled with the `engine` fixture.